docs/adr/XXXX-timezones.md

```diff
@@ -0,0 +1,77 @@
+# timezones
+
+* Status: proposed
+* Deciders: [Sander, Wouter, Arnaud, Bart]
+* Date: 2023-01-09
+
+Technical Story: [
+    https://github.com/inmanta/inmanta-core/issues/2919,
+    https://github.com/inmanta/inmanta-core/pull/2953,
+    https://github.com/inmanta/inmanta-lsm/pull/621,
+]
+
+## Context and Problem Statement
+
+How do we deal with timezones when it comes to datetime data? This data is passed in both directions over the API, stored at
+various places in the database and manipulated throughout the code. Which invariants do we uphold with regards to explicit
+timezone information for these datetime objects?
+
+## Decision Drivers
+
+* Behavior should be consistent regardless of the timezone participating hosts (e.g. server, agent, web-console client) live in.
+* Timezone handling over the API is non-trivial: we need to present a clear contract to the end-user.
+* In the absence of timezone information, there is no single sane interpretation: explicit timezones improve clarity.
+
+## Considered Options
+
+* Use UTC everywhere
+* Use timezone-aware datetimes everywhere
+* Support both timezone aware and naive timestamps as input, return naive timestamps. All naive timestamps are implicitly
+    assumed to be in UTC. Internally both a naive and an aware representation are considered.
+* Like the previous (mixed) option but make new end-points (where backwards compatibility is irrelevant) timezone-aware only.
+
+## Decision Outcome
+
+Chosen option: option 3 with timezone-aware internal representation, because it is as explicit as we can make (decision driver
```

Comment:
I would write it out in full, this is the most important part, so it should not have references (which may lead to confusion)